WordPress/Classes/ViewRelated/Activity/ActivityTableViewCell.swift

```diff
@@ -7,88 +8,56 @@ open class ActivityTableViewCell: WPTableViewCell {
 
     open override func awakeFromNib() {
         super.awakeFromNib()
-        assert(gravatarImageView != nil)
+        assert(iconBackgroundImageView != nil)
+        assert(contentLabel != nil)
         assert(summaryLabel != nil)
-        assert(statusImageView != nil)
-        assert(timestampLabel != nil)
-        assert(borderView != nil)
+        assert(rewindIcon != nil)
+        rewindIcon.image = rewindGridicon
+    }
+
+    override open func prepareForReuse() {
+        super.prepareForReuse()
+
+        iconImageView?.removeFromSuperview()
+        iconImageView = nil
```

Comment:
I'm not sure I understand why `iconImageView` is being recreated on reuse, can't we just set its image to nil?

Reply:
That view is added as a subview of the `iconBackgroundImageView` so we definitely need to remove it from there.